### PR TITLE
Configuring PWA for Mobile Browser

### PR DESCRIPTION
**Pull request by shimuldn**
Finxing PWA issue for Mobile Phones.
icon loading issue.

Before: 
Console error

Application error
No logo

Before

After this fix:

Install on firefox.
Install    Settings menu

We got logo!
Add to home screen now

I did checked on few browser and iPhone.
Looks good to me.

### CODE REVIEW

File: app/static/img/favicon/manifest.json [hunk 1]
```diff
@@ -1,41 +1,45 @@
 {
- "name": "App",
+ "name": "Whoogle Search",
+ "short_name": "Whoogle",
+ "display": "fullscreen",
+ "scope": "/",
+ "start_url": "$APP_ROOT/templates/index.html",
```
Comment (benbusby) [inline app/static/img/favicon/manifest.json:6]: Is this necessary? I don't think `APP_ROOT` is actually being used here, and it looks like everything else should work without a `start_url` property.

```suggestion
```

**Review by benbusby — COMMENTED** [review]
One small question/suggestion, otherwise looks good.

File: app/static/img/favicon/manifest.json (hunk 1 above)
Comment (shimuldn) [reply app/static/img/favicon/manifest.json:6]: I was looking with the sample https://developer.mozilla.org/en-US/docs/Web/Progressive_web_apps/Installable_PWAs
Yes, Work with `start_url` as well.

**Review by benbusby — APPROVED** [review]
Thanks, looks good.

**Pull request merged**